darts/utils/model_selection.py

```diff
@@ -0,0 +1,79 @@
+from typing import Sequence, Optional, Union
+from ..timeseries import TimeSeries
+
+
+def train_test_split(
+        data: Sequence[TimeSeries],
+        test_size: Optional[Union[float, int]] = 0.25,
+        axis: Optional[int] = 0,
+        n: Optional[int] = 0,
+        horizon: Optional[int] = 0,
+        ) -> Sequence[Sequence[TimeSeries]]:
```

Comment:
Should be `Tuple[Sequence[TimeSeries]]` ? Or actually even `Union[Tuple[TimeSeries], Tuple[Sequence[TimeSeries]]]` to support splitting simple `TimeSeries`.